packages/plugin-splash-screen/__tests__/generated.ts

```diff
@@ -79,7 +79,14 @@ describe("plugin-splash-screen", () => {
         );
       }
 
-      project.addPbxGroup([], "Resources", '""');
+      if (!project.findPBXGroupKey({ name: "Resources" })) {
```

Comment:
probably worth adding this to v13 as it's a safer check and adds it to the main group